Presentation.CardGamesGuruMiniApp/Program.cs

```diff
@@ -6,6 +9,13 @@
 builder.Services.AddControllersWithViews();
 builder.Services.AddConfig();
 
+//...
+builder.Services.AddMemoryCache();
+builder.Services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+builder.Services.AddReact();
+builder.Services.AddJsEngineSwitcher(options => options.DefaultEngineName = ChakraCoreJsEngine.EngineName).AddChakraCore();
```

Comment:
You can add all dependecies in services in the `Presentation.CardGamesGuruMiniApp.RunHelper.cs`, for stay `Program.cs `cleaner